### PR TITLE
LastFM stats updated ID and API key

### PR DESCRIPTION
**Pull request by skinner452**
# Description
After the last API key update, the app was no longer listed. This PR updates the ID to match the folder name so I don't need to worry about any mismatch. Also regenerated the API key again just in case.

# Copilot

copilot:all

### CODE REVIEW

File: apps/lastfmstats/manifest.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-id: lastfm-stats
+id: lastfmstats
```
Comment (matslina) [inline apps/lastfmstats/manifest.yaml:2]:
```suggestion
id: lastfm-stats
```

File: apps/lastfmstats/lastfmstats.star [hunk 2]
```diff
@@ -37,7 +37,7 @@ DEMO_TRACK = {
 
 # Decrypt the default API Key
 # Encrypted with `pixlet encrypt lastfmstats <apikey>`
-DEFAULT_API_KEY = secret.decrypt("AV6+xWcExxQCMGUycPsRYdeA7t7rOKr9SIvCfNyy+13/bcrin/XSYJOx5IqjoMsEe4LMS1x6tZz9cykLYglv5LPXIh9Lb2YKzIDZ4YqR3zOQLRH/ohMiocbld5kjCzXuSdDSSUwBXWgLJOm1yrW1IWrAGYPvvUID41LFuUzA7tZnKwUjpvg=")
+DEFAULT_API_KEY = secret.decrypt("AV6+xWcE/lRuZeOAg799Gnyb3QnTiPTpnGe4qg2I7DOIvuBWRg8QUBzyosDMDzV7HDhmCbz5Kx2LCMrhGFa2mb+rpxsLL5yyucpHtXVWiX4Gj+KrSfnFA3h2kId1yoW5+Csp1AN1sEcdZ38qq+26ItrNXCd8FcdGKA2p/YG9FvoyL6aFX6w=")
```
Comment (matslina) [inline apps/lastfmstats/lastfmstats.star:40]:
```suggestion
DEFAULT_API_KEY = secret.decrypt("AV6+xWcEMpic5Qa9aeMvuCkgUcDo+y8Ss5dDyiQPAQ7ulLP5TAWzms85w37wrjBimdEkp1lkVdBUwGt/gz1crVW7x1Z7LVCLCPCd5Cql/OofkaJeRDgYeCNKWPJtCRp22hER6a+CpFJShcjkVTEJVThJw9lK7ij0HrP5a3J9eHEXo10xv5M=")
```

**Review by matslina — COMMENTED** [review]
Hey. Changing the app ID will unfortunately break all existing installations, and generally cause mayhem behind the scenes in the production environment. The changes I proposed should repair the secrets, which as you wrote was encrypted using directory name instead of app ID as key.

**Comment by skinner452** [comment]
> Hey. Changing the app ID will unfortunately break all existing installations, and generally cause mayhem behind the scenes in the production environment. The changes I proposed should repair the secrets, which as you wrote was encrypted using directory name instead of app ID as key.

Ok, wasn't sure how that would work exactly. I committed the changes you proposed. Thank you!

**Comment by skinner452** [comment]
@matslina Friendly bump. Any other changes you are looking for?

**Comment by matslina** [comment]
@skinner452 Sorry for the delay! Another busy couple of weeks over here...

**Pull request merged**